src/rgw/driver/rados/rgw_datalog.h

```diff
@@ -242,8 +242,7 @@ class RGWDataChangesLog {
 
   const int num_shards;
   std::string get_prefix() {
-    auto prefix = cct->_conf->rgw_data_log_obj_prefix;
-    return prefix.empty() ? prefix : "data_log";
+    return cct->_conf->rgw_data_log_obj_prefix;
```

Comment:
```
src/rgw/driver/rados/rgw_datalog.h:245:24: error: no member named 'rgw_data_log_obj_prefix' in 'ConfigValues'
    return cct->_conf->rgw_data_log_obj_prefix;
           ~~~~~~~~~~~~^
```
i guess that should be `return "data_log"`?

Reply:
ops... sorry :grimacing: